src/components/AddHouse.jsx

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addHouse } from '../redux/house/addHouseSlice';
+import NavBar from './navBar/NavBar';
+
+export default function AddHouse() {
```

Comment:
- [ ] When I try to add a house to test the project's functionality, this happens on my end:

![image](https://user-images.githubusercontent.com/57421823/230063695-68dbc843-2295-4542-a22f-15982a8c2581.png)

It won't let me add a house, because the app is acting as if I'm not logged in.

Since I can't add houses, I can't test the "Delete" and "Favorites" functionality.  These features will be tested in the next review.

If your app requires special procedures to be able to be run correctly, kindly add them to your readme.  If this is a bug, please fix it so your app works as intended. 💪